runtime/src/main/java/org/corfudb/runtime/view/LayoutBuilder.java

```diff
@@ -201,7 +201,9 @@ public LayoutBuilder addLogunitServer(int stripeIndex, long globalLogTail,
      */
     public LayoutBuilder mergePreviousSegment(int segmentIndex) {
         if (segmentIndex < 1) {
-            throw new LayoutModificationException("No segments to merge.");
+            log.warn("mergePreviousSegment: No segments to merge.");
+            return this;
+            //throw new LayoutModificationException("No segments to merge.");
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine)

Reply:
Done.